api/openapi.yaml

```diff
@@ -388,3 +388,30 @@ paths:
       responses:
         204:
           description: OK
+
+  /tests/{testId}/version/{version}/definition.yaml:
+    get:
+      tags:
+        - api
+      parameters:
+        - in: path
+          name: testId
+          schema:
+            type: string
+            format: uuid
+          required: true
+        - in: path
+          name: version
+          schema:
+            type: integer
+          required: true
+      summary: Get the test definition as an YAML file
+      description: Get the test definition as an YAML file
+      operationId: getTestVersionDefinitionFile
+      responses:
+        200:
+          description: OK
+          content:
+            application/yaml:
+              schema:
+                $ref: "./definition_file.yaml#/components/schemas/Test"
```

Comment:
Maybe openapi is not the tool to define this, sine we're not actually using to transmit data to/from the server, but as a persistence file.  Adding to this, the naming conflicts with other existing entities.

I would leave this out of openapi, and just keep the format with autogenerated docs from the godocs in a `representation/yaml` package or somethig in that line.